core/src/test/scala/org/apache/spark/sql/test/TestConstants.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ *
+ * Copyright 2017 PingCAP, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.spark.sql.test
+
+object TestConstants {
+  val KeyTiDBAddress = "tidb.addr"
+  val KeyTiDBPort = "tidb.port"
+  val KeyTiDBUser = "tidb.user"
+  val KeyTestDB = "test.db"
+  val KeyTPCHDB = "tpch.db"
+  val KeyUseRawSparkMySql = "spark.use_raw_mysql"
+  val KeyMysqlAddress = "mysql.addr"
```

Comment:
It was my fault that in fact these config attributes with leading 'mysql' are used for possible future tests against raw mysql. So they are not used this moment, maybe we can add a comment here just in case?

Reply:
Or we can simply delete their usage if we are sure they won't be used anyhow.

Reply:
Agree.

Reply:
Done.